doc/supportedPlatformVersions.md

```diff
@@ -1117,6 +1120,7 @@
 - 8.2.9
 - 8.2.14
 - 8.2.16
+- 8.3.4
```

Comment:
do we need to add 8.3 for buster too? For latest node 20, seems like we skipped buster

Reply:
As of now, lets keep it.. from next release onwards lets decide.